Makefile

```diff
@@ -8,7 +8,7 @@ provider-dir-packer = packer/builders/$(env)/$(image)
 UNAME_S := $(shell uname -s)
 ifeq ($(UNAME_S),Linux)
 	docker_ssh_opts =  -e SSH_AUTH_SOCK=$(SSH_AUTH_SOCK) \
-	--volume $(SSH_AUTH_SOCK):$(SSH_AUTH_SOCK)
+	--volume $(SSH_AUTH_SOCK):$(SSH_AUTH_SOCK) --volume $(HOME)/.ssh/:/home/packer/.ssh/:ro
```

Comment:
Any trick here?

Reply:
I didn't understand why the mounting point for ssh folder wasn't configured for both (Darwin and Linux). In my computer (Linux) it doesn't work without configuring that.

` jhonatan@jmint  ~/workspace/packer-images-jhonatan   master  uname -s
Linux
`

Could you explain me?